ProtocolConformance.md

```diff
@@ -1,146 +1,294 @@
-Such set is immutable, and is not subject to replacement.
-
-If a protocol has no declared requirements, the protocol witness set for
-conformances to the protocol is empty.
+## 1.1 Creation of a Conformance
+A **protocol conformance** is the set of witnesses used by a concrete type to satisfy the requirements of a 
+protocol.  The syntactic declaration, `T: P`, that type `T` conforms to protocol `P` is separate and distinct 
+from the determination of the set of witnesses that is a conformance.  The substance of a conformance is not 
+declared.  It is created in response to a type being declared to conform to a protocol.  The set of witnesses 
+that is a conformance is inferred from the context.
+
+For a type `T` to conform to a protocol `P`, `T` must be declared to conform to `P`, and `T` must have at 
+least one implementation for each protocol requirement of `P`.  A named concrete type is declared to conform to 
+a protocol in one of two ways.  A non-generic concrete type is expressly declared to conform to a protocol.  A 
+concretization of a generic type is implicitly declared to conform to a protocol based on the pattern established 
+by the generic type's declaration and extensions.
+
+><sub>***Conretization of a Generic Type**
+>A concretization of a generic type is a specialized version of the generic type formed by replacing the generic 
+ type's type parameters with concrete type arguments.  A generic type may specify that its concretizations will 
+ conform, or conditionally may conform, to one or more protocols, which establishes a pattern for protocol 
+ conformance.  When a concretization is formed, its protocol conformances are determined by that pattern.*</sub>
+
+A declaration that `T` conforms to `P` further constitutes, with respect to each protocol `O`*<sub>i</sub>* which 
+`P` refines, a declaration that `T` conforms to `O`*<sub>i</sub>*.  Thus, for the declaration of `T: P` to be 
+valid, the implicit declarations of each `T: O`*<sub>i</sub>* must be valid.  For each conformance `T: O`*<sub>i</sub>*, 
+a distinct set of protocol witnesses is established. 
+
+A type `T` cannot be declared to conform to a protocol `P` if, within the visible scope, another declaration 
+exists of `T: P`.  This rule holds true even where competing declarations are conditional with disjoint conditions.  
+In a given scope, a type can conform to a protocol in only one way.
+
+
+## 1.2 Requirements
+
+A **protocol requirement** *m* is a statement in the declaration of a protocol that a type declared to conform 
+to the protocol must have a member satisfying *m*.
+
+If one protocol refines another protocol, the requirements of the latter are not part of the requirements of 
+the former.  The requirements of a protocol are not overridden by a protocol that refines the protocol.  Thus, 
+the conformances formed by a refining protocol do not include witnesses for requirements of the refined protocol.
```

Comment:
You said this 3 ways.  I'm not sure it *needs* to be said at all because it will be implied by a strict reading of the rest of the document, though I understand why you've included it: it was a point of confusion during discussions.  The right thing to do with stuff like that is to put it in a *note:*, which makes it “non-normative.”  I think it's important to separate the language *required* for understanding from the annotations and other supporting material.